multigeiger/webconf.cpp

```diff
@@ -7,41 +7,40 @@
 #include <HTTPClient.h>
 #include "userdefines.h"
 
-// All these flags are ON (=1) at startup
-// They all can be changed at any time using the web interface
-char speakerTick[2] = "1";
-char startSound[2] = "1";
-char ledTick[2] = "1";
-char showDisplay[2] = "1";
+bool speakerTick = (SPEAKER_TICK == 1);
```

Comment:
have SPEAKER_TICK as a boolean so this expression is not needed?

Reply:
?? SPEAKER_TICK is a #define, so it is an integer , isn't it?

Reply:
a #define is a preprocessor text replacement, so it can be a lot of things.